### PR TITLE
Update changelog for 6.0.0 release

### PR DESCRIPTION
**Pull request by corranwebster**
Also updates the copyright dates for 2023.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+pip-based installs of Enable significantly easier.
+
+Finally, as is usual for a release, there are a number of smaller bugfixes and
+minor improvements, such as bringing the PyCairo and Quartz backends back into
+a working state.
+
+Thanks to:
+
+* Mark Dickinson
+* Chengyu Lui
```
Comment (mdickinson) [inline CHANGES.txt:30]:
```suggestion
* Chengyu Liu
```

**Review by mdickinson — APPROVED** [review]
LGTM. I only reviewed the CHANGES.txt; I'm assuming that that was the only change aside from the copyright year updates?

**Comment by corranwebster** [comment]
Thanks, and yes, CHANGES.txt should be the only change other than the copyright updates.

**Comment by corranwebster** [comment]
Just that dispite telling Github to merge the suggestions, something failed and then I merged before I realized the problem.

I'll fix in the release PR.

**Pull request merged**